src/fides/ctl/core/export.py

```diff
@@ -458,11 +458,21 @@ def export_datamap(
             resource_type,
             exclude_keys=[],
         )
-        filtered_server_resources = [
-            resource
-            for resource in server_resources
-            if resource.organization_fides_key == organization_fides_key
-        ]
+
+        if not server_resources:
+            continue
```

Comment:
I'm assuming this is to trick mypy somehow?

Reply:
Right, I needed to prevent it from reaching the code below if there weren't any values to make mypy happy about the `None`, but I couldn't just `return` here because there could still be more resource types to check.